tools/packaging/packaging.strings.psd1

```diff
@@ -130,14 +130,14 @@ open {0}
         <version>{1}</version>
         <authors>Microsoft</authors>
         <owners>Microsoft,PowerShell</owners>
-        <requireLicenseAcceptance>true</requireLicenseAcceptance>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>Runtime for hosting PowerShell</description>
         <projectUrl>https://github.com/PowerShell/PowerShell</projectUrl>
-        <iconUrl>https://github.com/PowerShell/PowerShell/blob/master/assets/Powershell_black_64.png?raw=true</iconUrl>
-        <licenseUrl>https://github.com/PowerShell/PowerShell/blob/master/LICENSE.txt</licenseUrl>
+        <icon>{2}</icon>
+        <license type="expression">MIT</license>
         <tags>PowerShell</tags>
         <language>en-US</language>
-        <copyright>&#169; Microsoft Corporation.</copyright>
+        <copyright>&#169; Microsoft Corporation. All rights reserved.</copyright>
```

Comment:
"All rights reserved" was removed from these in <https://github.com/PowerShell/PowerShell/pull/12722>. Has the guidance changed?

Reply:
NuGet.org requires it now for Microsoft published packages.